resources/delete_from_list.rb

```diff
@@ -1,12 +1,12 @@
 property :path, String
```

Comment:
The tests are failing on delete_from_list.  Those are the most fragile and incomplete tests we have in this cookbook. I can look at the tests and failures tomorrow.  Have to deal with some other stuff the rest of today.